Sakura/AI/chat.py

```
@@ -0,0 +1,96 @@
+from Sakura import state
+
+def initialize_chat_client():
+    """Initialize OpenRouter client for chat"""
+    if OPENROUTER_API_KEY:
+        try:
+            state.openrouter_client = OpenAI(
+                base_url="https://openrouter.ai/api/v1",
+                api_key=OPENROUTER_API_KEY,
+            )
+            logger.info("✅ Chat client (OpenRouter) initialized successfully")
+        except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)